app/controllers/places_controller.rb

```diff
@@ -26,7 +26,7 @@ def edit
   # POST /places.json
   def create
     @place = Place.new(place_params)
-
+    @place.creator = current_user
```

Comment:
I suggest to do it the same way like in event controller: `@event = Event.new(event_params.merge(user_id: current_user.id))`

Reply:
When I have it like that: 
```
def create
    @place = Place.new(place_params.merge(user_id: current_user.id))
```

And I try to add new place I got message: 
Nowa miejscówka
1 error prohibited this place from being saved:
Creator musi istnieć

How to defined this ? (@place.creator ? )

Reply:
I think this is because of difference between column name and relation name. The best will be to debug it when it is running.